contributors/guide/contributor-cheatsheet/README.md

```diff
@@ -79,7 +79,6 @@ make your GitHub contribution experience better.
 - social@cncf.io - Contact the CNCF social team; blog, twitter account, and other social properties.
 - steering@kubernetes.io - Mail the steering committee. Public address with public archive.
 - steering-private@kubernetes.io - Mail the steering committee privately, for sensitive items.
-- helpdesk@rt.linuxfoundation.org - Mail the LF helpdesk for help with CLA issues.
```

Comment:
Can we instead add info about https://support.linuxfoundation.org?

Reply:
Hmm...it's not an email alias though :/

Reply:
I'll follow up with more explicit instructions on the CLA.

Reply:
@nikhita we can, but not in this section (it's not an email alias anymore).